index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,600;0,800;1,500&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="assets/css/styles.css">
+  <title>Welcome: I'm a Software Developer!</title>
+</head>
+<body>
+  <div class="homepage">
+    <header>
+      <nav>
+        <div class="nav-container">
+          <div class="logo">
+            <img src="assets/images/mobile/My-Logo.svg" alt="Logo">
+          </div>
+          <div class="menu-icon">
+            <img src="assets/images/mobile/Icon-Menu.svg" alt="Menue">
+          </div>
+        </div>
+      </nav>
+      <div class="header-container">
+        <ul>
+          <li><img src="assets/images/mobile/Rectangle7.svg" class="rectangle7" alt="image"></li>
+          <li><img src="assets/images/mobile/Ellipse5.svg" class="ellipse5" alt="image"></li>
+          <li><img src="assets/images/mobile/Rectangle5.svg" class="rectangle5" alt="image"></li>
+          <li><img src="assets/images//mobile//Ellipse4.svg" class="ellipse4" alt="ellipse4"></li>
+          <li><img src="assets/images/mobile/Ellipse3.svg" class="ellipse3" alt="image"></li>
+          <li><img src="assets/images/mobile/Ellipse13.svg" class="ellipse13" alt="image"></li>
+          <li><img src="assets/images/mobile/Ellipse12.svg" class="ellipse12" alt="image"></li>
+          <li><img src="assets/images/mobile/Rectangle6.svg" class="rectangle6" alt="image"></li>
+          <li><img src="assets/images/mobile/Ellipse11.svg" class="ellipse11" alt="image"></li>
+          <li><img src="assets/images/mobile/Ellipse8.svg" class="ellipse8" alt="image"></li>
+          <li><img src="assets/images/mobile/Ellipse14.svg" class="ellipse14" alt="image"></li>
+          <li><img src="assets/images/mobile/Ellipse15.svg" class="ellipse15" alt="image"></li>
+          <li><img src="assets/images/mobile/Rectangle4.svg" class="rectangle4" alt="image"></li>
+        </ul>
+
+        <div class="headline">
```

Comment:
- Your ``<div class="headline">`` should be inside a ``<section>``  tag and outside of your ``<header>``.